geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/IngestCommand.scala

```diff
@@ -84,6 +87,13 @@ trait IngestCommand[DS <: DataStore] extends DataStoreCommand[DS] {
     new ShapefileIngest(connection, Option(params.featureName), params.files, params.threads)
   }
 
+  protected def createJsonIngest(): Runnable = {
+    val filePath = params.files.head
+    val sft = GeoJsonInference.inferSft(filePath)
```

Comment:
maybe we should require them to specify the simple feature type name? or at least allow them to override the default

Reply:
Yeah there are a few options that could be user specified I just wasn't sure the best way to allow that. It'd be nice to allow specifying the id field too.

Would it make sense as a command line parameter? Or should we expect them to add it to the json somewhere?

Reply:
i think for ID we can just use an md5 of the row. but they can already specify the feature name on the command line, so we could just pass that in here. there's `-s` and `-f`, i think `-f` is only used to override the type name...